src/suit.c

```diff
@@ -84,8 +84,9 @@ static int suit_dry_run_manifest(struct suit_manifest_state *manifest_state,
 	for (enum suit_command_sequence seq = seq_name; seq <= end_seq; seq++) {
 		SUIT_DBG("Dry-run sequence: %d\r\n", seq);
 
-		struct zcbor_string *step_seq = suit_manifest_get_command_seq(manifest_state, seq);
-		if (step_seq == NULL) {
+		struct zcbor_string *step_seq = NULL;
+		ret = suit_manifest_get_command_seq(manifest_state, seq_name, &step_seq);
+		if (ret != SUIT_SUCCESS) {
```

Comment:
This return code may propagate and fail dry-run if the last sequence is not present?

Reply:
True, and what is more it did break the integration tests in sdk-nrf-next. Fixed.